ci/requirements-3.4.pip

```diff
@@ -1,3 +1,5 @@
-httplib2
-google-api-python-client
-oauth2client
+httplib2>=0.9.1
```

Comment:
the idea of the different pip files would be to different versions, so for sure these should NOT have an upper bound